server/lib/security/expense.ts

```diff
@@ -220,7 +220,7 @@ export const checkExpensesBatch = async (
       // Author Security Check: Checks if the author of the expense has 2FA enabled or not.
       addBooleanCheck(
         checks,
-        expense.User.hasTwoFactorAuthentication,
+        await expense.User.hasTwoFactorAuthentication(),
```

Comment:
We're creating an N+1 query issue here; loading 15 expenses will trigger 15 SQL queries. Could we use a loader for this?